src/components/application_manager/rpc_plugins/rc_rpc_plugin/test/commands/get_interior_vehicle_data_request_test.cc

```diff
@@ -242,7 +242,20 @@ TEST_F(
   (*mobile_message)[application_manager::strings::msg_params]
                    [message_params::kModuleType] = module_type;
   smart_objects::SmartObject radio_data;
+  smart_objects::SmartObject sis_data;
+  smart_objects::SmartObject gps_data;
+
+  const char kGpsShifted[] = "shifted";
```

Comment:
@mkorniichuk propose to move this constant to the file namespace. Otherwise it's better to rename it to `shifted_param` as for local variables we have a different naming rules